students/marc_charbo/lesson07/html_render.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+"""
+A class-based system for rendering html.
+"""
+
+class TextWrapper:
+    """
+    A simple wrapper that creates a class with a render method
+    for simple text
+    """
+    def __init__(self, text):
+        self.text = text
+
+    def render(self, file_out, current_ind=""):
+        file_out.write(current_ind)
+        file_out.write(self.text)
+
+# This is the framework for the base class
+class Element(object):
+    tag_start = '<html>'
+    tag_end = '</html>'
+
+    def __init__(self, content=None):
+        self.content = content
+        if self.content is None:
+            self.content = ''
+
+    def append(self, new_content):
+        self.new_content = new_content
+        self.content = self.content + ' ' + self.new_content
```

Comment:
Okay, i think this is what you were trying to reference.  You likely getting the error cause python things your trying to call the list function append first.  give this a different name like `append_content`. 
```
def append_content(self, new_content):
    self.content += new_content
```
Not sure why you are making a self.new_content variable here unless you are trying to use it somewhere else